cypress/e2e/landingPage.spec.cy.js

```diff
@@ -1,27 +1,128 @@
-/// <reference types="cypress" /> // is this really needed?
-import { onLandingPage } from "../support/pageObject/landingPage";
-import { onSignUpPage } from "../support/pageObject/signUpPage";
+import { faker } from "@faker-js/faker";
+// import { onLandingPage } from "../support/pageObject/landingPage";
+// import { onSignUpPage } from "../support/pageObject/signUpPage";
+
+const USERNAME = faker.person.firstName() + faker.number.int({ min: 1, max: 100 });
 
 describe("Start testing", () => {
-    before('open app', () => {
-        cy.visit('/');
-    })
-   it("sign up", () => { 
-   /*  cy.contains("Sign up").click();
-    cy.get('[placeholder="Username"]').clear().type('Igor963');
-    
-    cy.get('[placeholder="Email"]').clear().type('igor@yopmail.com');
-
-    cy.get('[placeholder="Password"]').clear().type('Test123@'); 
-
-    cy.get('button').click() */
-       onLandingPage.clickSignUpLink;
-      onSignUpPage.enterUserName;
-      onSignUpPage.enterEmail;
-      onSignUpPage.enterPSWD;
-      onSignUpPage.clickSignUpBtn; 
-      
-      //assert
-    }); 
+  beforeEach("open app", () => {
+    cy.visit("/");
+  });
+
+  // Add test with user name already taken
+  it("sign up", () => {
+    cy.contains("Sign up").click();
+    cy.get('[placeholder="Username"]').clear().type(USERNAME);
+
+    cy.get('[placeholder="Email"]')
+      .clear()
+      .type(USERNAME + "@yopmail.com");
```

Comment:
you can generate same mail with `faker` too by using `faker.internet.email()`

Reply:
i thought, i might need to receive an email and get some values from there. If not, it's not difficult to introduce changes